src/app/util/mock/MockNodeConfig.h

```diff
@@ -74,11 +74,18 @@ struct MockEventConfig
     const EventId id;
 };
 
+enum class MockClusterSide : uint8_t
+{
+    kServer = 0x1,
+    kClient = 0x2,
+};
+
 struct MockClusterConfig
 {
     MockClusterConfig(ClusterId aId, std::initializer_list<MockAttributeConfig> aAttributes = {},
                       std::initializer_list<MockEventConfig> aEvents = {}, std::initializer_list<CommandId> aAcceptedCommands = {},
-                      std::initializer_list<CommandId> aGeneratedCommands = {});
+                      std::initializer_list<CommandId> aGeneratedCommands = {},
+                      BitMask<MockClusterSide> side = BitMask<MockClusterSide>().Set(MockClusterSide::kServer));
```

Comment:
is a mask valid? Can clusters be both client and server at the same time?
Why is this not a `MockClusterSide` ? This should be explained in a comment if we really are to support a bit mask.

Reply:
Yes, I think it should be valid, as the OnOffLightSwith should have both client and server of identify cluster.